src/components/Workshop/Workshop.js

```diff
@@ -1,60 +1,139 @@
-import React from 'react'
-import { object } from 'prop-types'
+import React, { useState } from 'react'
+import { object, node, number } from 'prop-types'
 import Card from '@material-ui/core/Card'
 import CardContent from '@material-ui/core/CardContent'
 import Divider from '@material-ui/core/Divider'
 import ReactMarkdown from 'react-markdown'
+import AppBar from '@material-ui/core/AppBar'
+import Tab from '@material-ui/core/Tab'
+import Tabs from '@material-ui/core/Tabs'
 
-import { recipesMap } from '../../data/maps'
+import { features } from '../../config'
+import { recipeType } from '../../enums'
+import { forgeRecipesMap, kitchenRecipesMap, recipesMap } from '../../data/maps'
 import Recipe from '../Recipe'
 
 import FarmhandContext from '../../Farmhand.context'
 
 import './Workshop.sass'
 
-const Workshop = ({ learnedRecipes }) => (
-  <div className="Workshop">
-    <h3>
-      Learned Recipes ({Object.keys(learnedRecipes).length} /{' '}
-      {Object.keys(recipesMap).length})
-    </h3>
-    <section>
-      <ul className="card-list">
-        {Object.keys(learnedRecipes).map(recipeId => (
-          <li key={recipeId}>
-            <Recipe
-              {...{
-                recipe: recipesMap[recipeId],
-              }}
-            />
-          </li>
-        ))}
-      </ul>
+function TabPanel(props) {
+  const { children, value, index, ...other } = props
+
+  return (
+    <section
+      role="tabpanel"
+      hidden={value !== index}
+      id={`workshop-tabpanel-${index}`}
+      aria-labelledby={`workshop-tab-${index}`}
+      {...other}
+    >
+      {value === index && children}
     </section>
-    <Divider />
-    <section>
-      <ul className="card-list">
-        {/*
+  )
+}
+
+TabPanel.propTypes = {
+  children: node,
+  index: number.isRequired,
+  value: number.isRequired,
+}
+
+const a11yProps = index => ({
+  id: `workshop-tab-${index}`,
+  'aria-controls': `workshop-tabpanel-${index}`,
+})
+
+const Workshop = ({ learnedRecipes }) => {
+  const [currentTab, setCurrentTab] = useState(0)
+
+  const learnedKitchenRecipes = []
+  const learnedForgeRecipes = []
+
+  for (const learnedRecipeId of Object.keys(learnedRecipes)) {
```

Comment:
this would make a nice little utility helper to break out as well. i actually wouldn't mind two (`getLearnedKitchenRecipes` and `getLearnedForgedRecipes`) but i see the appeal of a singular loop too. either way it seems like it could be plucked out of this component pretty easily